.eslintrc.js

```diff
@@ -25,5 +25,6 @@ module.exports = {
       "warn",
       { argsIgnorePattern: "^_", varsIgnorePattern: "^_" },
     ],
+    "@typescript-eslint/consistent-type-exports": "error"
```

Comment:
https://typescript-eslint.io/rules/consistent-type-exports

Reply:
OMG!!! Us, too!! Oh.. but for imports. Ha, i guess having this would be nice as well.
I would add the side effects and imports rules as well. 
```suggestion
    "@typescript-eslint/consistent-type-exports": "error",
    "@typescript-eslint/no-import-type-side-effects": "error",
    "@typescript-eslint/consistent-type-imports": "error"
```

Here is my OG pr description with some links to why this is a thing, and a new imports plugin helper thing. I would perhaps adopt the plugin here as well. 

> **PR Description**
> Had a very strange bug on main today that made it so I couldn't run the app in dev mode!! Because of some typing shenanigans a Component and a type were both named Headlines. The error didn't show up with tsc or eslint, or during build. But during dev, vite got super grumpy. So then I found some new TS config values and new eslint rules to protect against this! This will help make it easier to continue sharing types and code throughout our code base.

> Oh, and all of this is auto fixable except the weird Headlines thing which was easy to fix once the squiggles were on line

> https://typescript-eslint.io/blog/consistent-type-imports-and-exports-why-and-how/
> https://devblogs.microsoft.com/typescript/announcing-typescript-5-0-beta/#verbatimmodulesyntax

> **Other Notes**
> I found the strangest way to install a package in npm, but it is [the suggested way from their docs](https://github.com/un-es/eslint-plugin-i#installation)